openhands/runtime/utils/edit.py

```diff
@@ -121,7 +121,12 @@ def __init__(self, enable_llm_editor: bool, *args: Any, **kwargs: Any) -> None:
             )
             draft_editor_config.caching_prompt = False
 
-        self.draft_editor_llm = LLM(draft_editor_config, metrics=llm_metrics)
+        self.draft_editor_llm = LLM(
```

Comment:
@openhands modify this so that `draft_editor_llm` is passed into the __init__ method here instead of `draft_editor_config`. Modify any callers to use the new constructor